src/earth/Earth.test.js

```diff
@@ -5,8 +5,8 @@ import Earth from './Earth';
 const toi = createTimeOfInterest.fromTime(2017, 12, 10, 0, 0, 0);
 const earth = new Earth(toi);
 
-it('tests getName', () => {
-    expect(earth.getName()).toBe('earth');
+it('tests if name is correct', () => {
```

Comment:
`tests if name is correct` -> this style of test case descriptions is OK, it properly reflects the assertion made in the test.

I'm used to define the descriptions like `it('get name should return the expected value')`, but this choice is quite subjective and is a matter of taste.

Reply:
Yep. I like to read it like a sentence "it get name should..." does not fit in that case ;)

Reply:
OK.

Another question about the tests: is there a special reason to implement the tests for this project in JS instead of TS?

If there is no specific reason for that, the tests can be easily converted into `*.spec.ts` files.

The Jest testing framework can also be used with TypeScript, only one additional dev dependency is required (`@types/jest`).

Also, sometimes it is useful to group related test cases inside `describe` calls, for example

```javascript
describe('myFunction should', () => {
  it('throw an error is the arg is falsy', () => {
    ...
  });

  it('return expected value', () => {
    ...
  });
}
```

This nesting results in a more readable formatting of the test output/reports.

Reply:
oups sorry, I've just seen that `describe` is already used in some of the existing suites.

Reply:
There is no specific reason for using JS for the tests. Would say its a historical thing. TS was introduces after I introduces unit tests. I am open to change that